sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveQuerySuite.scala

```diff
@@ -73,9 +72,17 @@ class HiveQuerySuite extends HiveComparisonTest with SQLTestUtils with BeforeAnd
     }
   }
 
-  private def assertUnsupportedFeature(body: => Unit): Unit = {
-    val e = intercept[ParseException] { body }
-    assert(e.getMessage.toLowerCase(Locale.ROOT).contains("operation not allowed"))
+  private def assertUnsupportedFeature(
+    body: => Unit,
+    message: String,
+    expectedContext: ExpectedContext): Unit = {
```

Comment:
4 spaces, see https://github.com/databricks/scala-style-guide#spacing-and-indentation:
```suggestion
      body: => Unit,
      message: String,
      expectedContext: ExpectedContext): Unit = {
```